(Doc+) Resolve watermark on Cloud via Autoscaling (#115990)
elastic/elasticsearch: docs/reference/troubleshooting/common-issues/disk-usage-exceeded.asciidoc

to your use case: 

* add nodes to the affected <<data-tiers,data tiers>>
+

> TIP: We recommend enabling <<xpack-autoscaling,Autoscaling>> for clusters deployed via our {ess}, {ece}, and {eck} platforms.

Reviewer: ```suggestion
TIP: You should enable <<xpack-autoscaling,autoscaling>> for clusters deployed using our {ess}, {ece}, and {eck} platforms.
```
